src/components/editor-page/document-bar/document-info/document-info-line-word-count.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import React, { useEffect, useState } from 'react'
+import { Trans, useTranslation } from 'react-i18next'
+import { ShowIf } from '../../../common/show-if/show-if'
+import { DocumentInfoLine } from './document-info-line'
+import { UnitalicBoldText } from './unitalic-bold-text'
+import { useIFrameEditorToRendererCommunicator } from '../../render-context/iframe-editor-to-renderer-communicator-context-provider'
+
+export const DocumentInfoLineWordCount: React.FC = () => {
```

Comment:
I started to add js docs to my components. Maybe you want to jump on that train, too? :yum:

Reply:
done

Reply:
However, we should create a PR adding specific ESDoc conventions for our project (for example in the contribution guide).